client/src/App.vue

```diff
@@ -23,6 +23,7 @@
         :redirectOnLoginSuccess="getNameRequestUrl"
         :redirect-on-login-fail="getNameRequestUrl"
         :redirect-on-logout="getNameRequestUrl"
+        :show-actions="false"
```

Comment:
So, technically it's an enable prop, not a disable prop 😛 

(No change required! Just ribbing ya.)

Reply:
lol ya got me! 

It was initially a disable prop... many many moons ago 😆